iquip/apps/builder.py

```diff
@@ -574,6 +582,110 @@ def scanArguments(self) -> Dict[str, Any]:
         }
 
 
+class _CenterScan(_BaseScan):
+    """Widget for center scan in _ScanEntry.
+
+    Attributes:
+        centerSpinBox: QDoubleSpinBox for center argument inside state.
+        spanSpinBox: QDoubleSpinBox for span argument inside state.
+        stepSpinBox: QDoubleSpinBox for step argument inside state.
+        randomizeCheckBox: QCheckBox for randomize argument inside state.
+    """
+    def __init__(
```

Comment:
Oh, I didn't realize that there is no empty line between the class definition and the first method.
It's the same situation for `_BaseScan` and other scan classes, so please fix it.

Reply:
And in line 490, the closing parenthesis of `_NoScan.__init__()` has a wrong indentation.
Please check it.